docs/data/material/components/accordion/AccordionExpandDefault.js

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import Accordion from '@mui/material/Accordion';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import Typography from '@mui/material/Typography';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+
+export default function AccordionExpandDefault() {
+  return (
+    <div>
+      <Accordion defaultExpanded>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel1-content"
+          id="panel1-header"
+        >
+          <Typography>Exapnded by default</Typography>
```

Comment:
```suggestion
          <Typography>Expanded by default</Typography>
```

Reply:
Jeez, why it's so hard to write this word correctly 😅